docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:8u372-b07-jre-jammy
+FROM eclipse-temurin:17.0.14_7-jre-jammy
```

Comment:
It’s deployed within our VPC and it doesn’t access network outside of VPC. Do we still need to take care of vulnerabilities?